solidity/contracts/token/fees/visualize_fees.py

```
@@ -0,0 +1,274 @@
+    parser.add_argument('--maxfee', type=float, default=1000,
+                       help='Maximum fee in token units')
+    parser.add_argument('--halfamount', type=float, default=10000,
+                       help='Amount at which fee equals maxFee/2')
+    parser.add_argument('--maxamount', type=float, default=50000,
+                       help='Maximum amount to visualize')
+    parser.add_argument('--output', type=str, default='fee_curves',
+                       help='Output file prefix (without extension)')
+
+    args = parser.parse_args()
+
+    print(f'\nGenerating fee curve visualizations...')
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)